pkgs/by-name/sp/sptlrx/package.nix

```
@@ -2,19 +2,25 @@
 
 buildGoModule rec {
   pname = "sptlrx";
-  version = "1.1.0";
+  version = "1.2.2";
 
   src = fetchFromGitHub {
     owner = "raitonoberu";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-6GbefTWrhH6RdASmSrugd4xESkwqFVF5qwFmf0JUDTY=";
+    hash = "sha256-b8ALhEjolH0RH+I9HVQeOagPBi2isLNUxqKdj5u2O9s=";
   };
 
-  vendorHash = "sha256-Ll5jUjpx4165BAE86/z95i4xa8fdKlfxqrUc/gDLqJ0=";
+  vendorHash = "sha256-pExSQcYjqliZZg/91t52yk6UJ4QCbpToMpONIFUNkwc=";
 
   ldflags = [ "-s" "-w" ];
 
+  checkFlags =
+    let
+      skippedTests = [ "TestGetIndex" ];
```

```suggestion
      # Requires network access
      skippedTests = [ "TestGetIndex" ];
```
It's considered good practice to leave a comment about why tests are disabled so it's easier for other people making changes in the future. Not a blocker, though